contracts/RewardsDistributer.sol

```diff
@@ -142,6 +142,32 @@ contract RewardsDistributer is IRewardsDistributer, Initializable, OwnableUpgrad
         }
     }
 
+    /**
+     * @notice Reset era reward remove table.
+     * Only Owner can call.
+     * @param indexers address[]
+     * @param eras uint256[]
+     * @param addAmounts uint256[]
+     * @param removeAmounts uint256[]
+     */
+    function resetEraRewardRemoveTables(
+        address[] memory indexers,
+        uint256[] memory eras,
```

Comment:
Shoud `uint256[] memory eras,` be era number other than era array?

Reply:
use `array` to batch many operations in different eras